.github/workflows/release.yml

```diff
@@ -199,8 +198,8 @@ jobs:
 
       - name: Setup git
         run: |
-          git config user.email "maintainers-python+github@nuxeo.com"
-          git config user.name "GitHub-CI action (Release)"
+          git config user.email "drive@hyland.com"
```

Comment:
use these as var, as they are used in several places.

Reply:
Done